Graphics/Implicit/ObjectUtil/GetImplicitShared.hs

```diff
@@ -56,7 +56,7 @@ getImplicitShared ctx (UnionR _ []) =
 getImplicitShared ctx (UnionR r symbObjs) = \p ->
   rminimum r $ fmap (flip (getImplicit' ctx) p) symbObjs
 getImplicitShared ctx (IntersectR _ []) =
-  getImplicitShared @obj ctx Full
+  getImplicitShared @obj ctx Empty
```

Comment:
is this just wrong?

Reply:
I think so, based on https://en.wikipedia.org/wiki/Intersection_(set_theory)#Algebraic_properties 

> The intersection of any set with the [empty set](https://en.wikipedia.org/wiki/Empty_set) results in the empty set;

Reply:
Also our implicit space (or starting object?) seems to be `emptySpace` - if we have `union [oneObject]` or just `oneObject` (without implicit union xD) it seems like it can only be unioned with `emptySpace`, because if it was a `fullSpace` we would get `fullSpace` instead of `oneObject`.